rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml

```diff
@@ -0,0 +1,34 @@
+title: Lolbin Ssh.exe Use As Proxy
+id: 7d6d30b8-5b91-4b90-a891-46cccaf29598
+status: experimental
+description: Detect use of ssh.exe as proxy to launch another program
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Ssh/
+author: frack113
+date: 2022/12/29
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_ssh_img:
+        - Image|endswith: '\ssh.exe'
+        - ImageFileName: 'ssh.exe'
+    selection_ssh_cmd: 
+        CommandLine|contains: 'localhost'
+        CommandLine|endswith: '.exe'
```

Comment:
For this, I suspect that you can also do it with `127.0.0.1`  and your IP for example `192.168.1.11`. Also, is the `.exe` needed? Can't you do something like `ssh localhost calc`

Reply:
Yes , it is a mess to catch it .Let's focus on suspicius ParentImage only

Reply:
Yes. That's the best bet and we tune legitimate use accordingly.